.gitlab-ci.yml

```diff
@@ -1032,6 +1056,14 @@ test_full_integration:
     reports:
       junit: results_full_integration.xml
 
+test_full_integration_open:
```

Comment:
Suggestion:
Same as for the backend tests. Share code in a job named: `test_full_integration`, and then extend them in:
`test_full_integration:open_source`
and
`test_full_integration:enterprise`